src/commands/new/dto.ts

```diff
@@ -0,0 +1,46 @@
+import { GluegunToolbox } from 'gluegun';
+import { OptionAttribute } from '../../extensions/args-extension';
+
+const DTO_OPTIONS: OptionAttribute[] = [{ name: 'name', alias: 'n', required: true }];
+
+module.exports = {
+  name: 'dto',
+  options: DTO_OPTIONS,
+  description: 'Generates a new SOA Remote DTO',
+  run: async (toolbox: GluegunToolbox) => {
+    const {
+      print,
+      parameters,
+      template: { generate },
+      args,
+      utils
+    } = toolbox;
+
+    try {
+      const options = args.parseOptions(DTO_OPTIONS);
+      const outputDirectory = args.getOutputDirectory();
+      const componentType = parameters.command;
+      const componentName = options.name;
+      const className = `${componentName}RemoteDto`;
+
+      const props = {
+        componentName
+      };
+
+      toolbox.parameters.options.name = className;
+      const metaCommand = utils.getCommand('meta');
+      const testCommand = utils.getCommand('test');
+
+      metaCommand.run(toolbox);
+      testCommand.run(toolbox);
+
+      await generate({
+        template: `${componentType}.cls.ejs`,
+        target: `./${outputDirectory}/${className}.cls`,
+        props
+      });
+    } catch (err) {
+      print.error(print.colors.error(err.message));
```

Comment:
Add `process.exit(1)` to have the CLI exit after the error.
```suggestion
      print.error(print.colors.error(err.message));
      process.exit(1);
```